templates/vars/demo.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
I expect you need an empty hub vnet id parameter as well

Reply:
This file wasn't there so I just added this file but need to see how the demo environment is being created without this file.